Gemfile.lock

```diff
@@ -72,6 +72,8 @@ GEM
       rake (>= 10.0.0)
     sass-embedded (1.57.1-arm64-darwin)
       google-protobuf (~> 3.21)
+    sass-embedded (1.57.1-x64-mingw-ucrt)
+      google-protobuf (~> 3.21)
```

Comment:
why there is a change in gemfile

Reply:
I think it happened during Jekyll's updation on my local machine. I mistakenly committed that without reverting.